src/digest/crc32.cr

```diff
@@ -17,16 +17,16 @@ class Digest::CRC32 < ::Digest
     LibZ.crc32(0, nil, 0).to_u32
   end
 
-  def self.checksum(data) : UInt32
+  def self.checksum(data : String) : UInt32
```

Comment:
Even this constraint is too strict: it calls `#update` internally.

Reply:
Ack, good catch, will remove